Makefile

```diff
@@ -28,12 +29,9 @@ ifndef GCE_PD_CSI_STAGING_VERSION
 endif
 	go build -ldflags "-X main.vendorVersion=${STAGINGVERSION}" -o bin/${DRIVERBINARY} ./cmd/
 
-build-windows:
+gce-pd-driver-windows:
 	mkdir -p bin
-ifndef GCE_PD_CSI_STAGING_VERSION
-	$(error "Must set environment variable GCE_PD_CSI_STAGING_VERSION to staging version")
-endif
-	GOOS=windows go build -ldflags "-X main.vendorVersion=${STAGINGVERSION}" -o bin/${DRIVERWINDOWSBINARY} ./cmd/
+	GOOS=windows go build -ldflags -X=main.vendorVersion=$(REV) -o bin/${DRIVERWINDOWSBINARY} ./cmd/
```

Comment:
How come we use REV instead of STAGINGVERSION like the rest of the rules?

Reply:
we changed it to use a default value getting from git